cartographer/mapping/2d/tsdf_range_data_inserter_2d_test.cc

```diff
@@ -48,7 +48,7 @@ class RangeDataInserterTest2DTSDF : public ::testing::Test {
   }
 
   void InsertPoint() {
-    sensor::RangeData range_data;
+    sensor::RangeData range_data = sensor::RangeData();
```

Comment:
if initialization is the issue, this won't fix it. We could either add a `CreateRangeData()` function which returns a zero-initialized RangeData (I assume the origin is the issue) or add a constructor to the struct (and then maybe make it a class).

Reply:
@kdaun Just realized that since C++11 this actually forces zero initialization, so you're right ...